MetaMask/metamask-mobile
# chore: update Android SDK to API Level 34

## **Description**

Bump target Android API Level to 34 and address crashes reported [here](https://github.com/MetaMask/metamask-mobile/issues/10352) that were introduced by [this PR](https://github.com/MetaMask/metamask-mobile/pull/10282).




## **Related issues**

Fixes: #10352 and #10260

## **Manual testing steps**

1. Install MM app
2. Import SRP wallet
3. Click Done
4. App should not crash

## **Screenshots/Recordings**



### **Before**

https://github.com/user-attachments/assets/f36d5705-cc29-4e6d-9f17-75a5888d563c

### **After**

https://github.com/user-attachments/assets/38118432-2e7d-4777-9ad2-3fe05fe8f392


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
